README.md

```diff
@@ -1,2 +1,57 @@
-# ods-e2e-tests
-End-to-end functional tests, based on Spock/Geb and gradle to assert the correct functionality of an OpenDevStack installation
+# Automated test cases for ODS
+
+## Description
+Suite of test cases for OpenDevStack and the Release Manager.
+The build is setup to work with Chrome, Chrome headless and html. Have a look at the `build.gradle` and the `src/test/resources/GebConfig.groovy` files.
+
+It contains too the Firefox driver that it is not fully tested as currently the provissioning app does not work with Firefox
```

Comment:
@jorge-romero There are some spelling and grammatical errors. Let's go over this in an additional commit.

Reply:
Yes, thanks I have to review it...